src/Common/src/TypeSystem/Common/DefType.FieldLayout.cs

```diff
@@ -230,6 +230,14 @@ public int ThreadStaticFieldAlignment
             }
         }
 
+        /// <summary>
+        /// Do the fields of the type satisfy the Homogeneous Float Aggregate classification on ARM architecture?
+        /// </summary>
+        public virtual bool IsHFA()
```

Comment:
Its part of the layout of a field that needs to be computed by the field layout algorithm for some platforms. (ARM, and probably also ARM64) I'm ok with it here.